src/liquid/components/ld-bg-cells/ld-bg-cells.tsx

```diff
@@ -22,18 +23,22 @@ export type CellType =
 @Component({
   assetsDirs: ['assets'],
   tag: 'ld-bg-cells',
-  styleUrl: 'ld-bg-cells.shadow.css',
+  styleUrl: 'ld-bg-cells.css',
   shadow: true,
 })
 export class LdBgCells {
   /** Cells pattern */
-  @Prop() type: CellType = 'safc'
+  @Prop() type: CellType = 'hexagon'
 
   render() {
-    const assetPath = getLdAssetPath(`./assets/${this.type}-cell.svg`)
+    // Handle aliases (for backward compatibility).
+    let cellType = this.type
+    if (cellType === 'qa-x2f-qc') cellType = 'supelco'
```

Comment:
```suggestion
    const cellType = this.type === 'qa-x2f-qc' ? 'supelco' : this.type
```

Reply:
I'll keep it the way it is. It's readable and allows adding more aliases in the future via copy pasting the if line.